requirements/requirements.txt

```diff
@@ -1 +1 @@
-jumanji==0.3.1
+jumanji==1.0.1
```

Comment:
Planning on releasing a new version of jumanji so can we just unpin this package :pray: 
```suggestion
jumanji
```

Reply:
Thankfully! In future we can adjust both mava and matrax when it's needed then 🥳